src/test/java/com/example/moviemanager/controller/MovieControllerTest.java

```diff
@@ -37,129 +49,215 @@ public class MovieControllerTest {
     @Autowired
     private ObjectMapper objectMapper;
 
+    @ParameterizedTest
+    @CsvFileSource(resources = "/validation_test.csv", numLinesToSkip = 1)
```

Comment:
I like that you created such test. One thing I would improve is csv file name. Express what it is meant for invalid_create_movie_request_params.csv